src/components/ItaliaTheme/Blocks/Listing/CardWithSlideUpTextTemplate.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Container,
+  CardReadMore,
+} from 'design-react-kit/dist/design-react-kit';
+import { UniversalLink } from '@plone/volto/components';
+import { flattenToAppURL } from '@plone/volto/helpers';
+import { getCalendarDate } from '@italia/helpers';
+import {
+  ListingLinkMore,
+  ListingCategory,
+} from '@italia/components/ItaliaTheme';
+import cx from 'classnames';
+import { getCategory } from '@italia/components/ItaliaTheme/Blocks/Listing/Commons/utils';
+import { defineMessages, useIntl } from 'react-intl';
+
+const messages = defineMessages({
+  vedi: {
+    id: 'card_vedi',
+    defaultMessage: 'Vedi',
+  },
+});
+
+const CardWithSlideUpTextTemplate = ({
+  items,
+  title,
+  isEditMode,
+  linkTitle,
+  linkHref,
+  show_block_bg = false,
+  always_show_image = true,
+  show_type = true,
+  show_section,
+  show_description = true,
+  hide_dates = false,
+}) => {
+  const intl = useIntl();
+
+  return (
+    <div className="card-slide-text-template">
+      <Container>
+        <div className="title">{title && <h2>{title}</h2>}</div>
+        <div className="grid mb-3 mt-5">
+          {items.map((item, index) => {
+            let image = item?.image || item?.immagine_testata;
+            const category = getCategory(item, show_type, show_section);
+            const date = hide_dates ? null : getCalendarDate(item);
+
+            return (
+              <UniversalLink
+                item={!isEditMode ? item : null}
+                href={isEditMode ? '#' : null}
+                style={{
+                  backgroundImage: `url(${flattenToAppURL(
+                    image?.scales?.preview?.download || '',
+                  )})`,
+                }}
+                className={cx('listing-item box bg-img', {
+                  'no-image': !always_show_image,
```

Comment:
Toglierei l'opzione 'mostra l'immagine per tutti gli elementi' (anche nella sidebar) perchè in questo template ha poco senso.